swat/base.py

```diff
@@ -154,6 +154,8 @@ def remove(self, key: str) -> bool:
 
     def get(self, key: str, validate_type: Optional[Literal['oauth', 'service']] = None,
             missing_error: bool = True) -> Optional[Cred]:
+        if key not in self.store:
```

Comment:
This method returned `Optional[Cred]`, which meant that `None` was valid, and on the caller to handle (as opposed to raising a missing key error)